### ISSUE TITLE
Change default google drive permissions

### ISSUE THREAD

**Issue by bryancross** [bryancross/key2pdf#14]
Change permissions so that GitHub users do not have to ask permission to view uploaded files.

**Comment by tcbyrd** [comment]
@bryancross Try this link: https://goo.gl/WezWB7

Instead of changing permissions in the script, I created a folder in Drive that anyone at GitHub can edit:
https://drive.google.com/drive/folders/0Bx7UhUK81Pz1SE9YZ1JUR0FPbmc

Then added the [`parents`](https://developers.google.com/drive/v3/web/folder) key to the script to tell it to upload to that folder:
```
  service.files.create({
    resource: {
      name: file.name,
      mimeType: 'application/pdf',
      parents: ['0Bx7UhUK81Pz1SE9YZ1JUR0FPbmc']
```

Permissions can then be controlled using the Google Drive UI without having to modify the script. All the script knows is the ID of the folder it's supposed to store it in.

Thoughts on this implementation? We could add a config variable for the Google Drive folder if we wanted to make it configurable.

**Comment by tcbyrd** [comment]
Hm this didn't get closed in #17 when you merged it, so I'm closing it now! 🚫
